src/scss/index.scss

```diff
@@ -1 +1,109 @@
+@import "variables";
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
 
+$enable-shadows: true;
+
+
+.btn {
+  text-decoration: none;
```

Comment:
I'm a bit surprised this is necessary, I think this is a default in BS

Reply:
Oh, you are absolutely right!